build/containers/Dockerfile.anomaly-detection-app

```diff
@@ -9,7 +9,7 @@ COPY ./samples/apps/anomaly-detection-app .
 
 RUN echo "Creating container based on ${PLATFORM}/debian:bullseye-slim" && \
     apt-get update && \
-    apt-get install -y protobuf-compiler libprotoc-dev python3-pip && \
+    apt-get install -y protobuf-compiler libprotoc-dev python3-pip python3-sklearn && \
```

Comment:
This makes the app to use system wide `scikit_learn` as well as system wide `numpy`, for bullseye none of these two will be the version we had in requirements.txt

Reply:
that's correct, arm32v7 build stuck forever if use the version specified in requirements for numpy and scikit_learn.